lib/ask_export/survey_response_fetcher.rb

```diff
@@ -15,7 +15,9 @@ def initialize(since_time, until_time)
     end
 
     def call
-      raise "Too early, submissions for today are still open" if until_time > Time.zone.now
+      if until_time > Time.zone.now
+        raise ArgumentError, "Too early, submissions for today are still open"
```

Comment:
The copy is still the same, which conflicts with the commit message. Looks like this change is actually in the next commit. Maybe just combine the commits?

Reply:
Oops, thanks 👍